app/views/transactions/index.html.erb

```diff
@@ -1,13 +1,13 @@
 <%= content_for :current_page do %>
-  <span><a href="/">Transactions</a></span>
+  <span><a href="/">Transaction Activities</a></span>
```

Comment:
I hate to say it, but we may get push back here from DC.  I know that calling this Transaction Activities makes sense from a back end perspective, but from the DC perspective they are used to seeing these records as Transactions and will probably want an explanation.  Speaking from past experience, they are sensitive to any UI changes that weren't specifically requested.